docker/tag.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+
+# This script is used by cloud build to push Docker images into Docker hub
+
+tag_and_push() {
+    tag=$1
+    docker tag gcr.io/$PROJECT_ID/graph-node:$SHORT_SHA \
+           lutter/graph-node:$tag
+    docker push lutter/graph-node:$tag
+
+    docker tag gcr.io/$PROJECT_ID/graph-node-debug:$SHORT_SHA \
+           lutter/graph-node-debug:$tag
+    docker push lutter/graph-node-debug:$tag
+}
+
+echo "Logging into Docker Hub"
+echo $PASSWORD | docker login --username=lutter --password-stdin
```

Comment:
Same questions here regarding the use `lutter` and using a variable for the username.

Reply:
This one unfortunately has to stay, or, at least be _some_ user rather than just an organization; AFAICT, Docker Hub can only generate tokens for users, not for orgs and you have to log in as that user.